lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -186,10 +189,12 @@ def write_ssh_key
       def start_ansible_runner
         env = {}
```

Comment:
As mentioned in https://github.com/theforeman/puppet-foreman_proxy/pull/777#discussion_r977536686 I wonder if we should present everything as environment variables by default. In that case I'd probably extract the env var construction to a separate method.